modules/tinymce/src/themes/silver/main/ts/ui/dialog/Collection.ts

```diff
@@ -57,13 +77,14 @@ export const renderCollection = (
       const ariaLabel = itemText.replace(/\_| \- |\-/g, (match) => mapItemName[match]);
 
       const disabledClass = providersBackstage.isDisabled() ? ' tox-collection__item--state-disabled' : '';
-      return `<div class="tox-collection__item${disabledClass}" tabindex="-1" data-collection-item-value="${Entities.encodeAllRaw(item.value)}" title="${ariaLabel}" aria-label="${ariaLabel}">${iconContent}${textContent}</div>`;
+      return `<div class="tox-collection__item${disabledClass}" tabindex="-1" data-collection-item-value="${Entities.encodeAllRaw(item.value)}"` +
```

Comment:
This is getting really hard to read. I wonder if we can break it up into an array that joins.